tools/jsii-compliance/report.ts

```diff
@@ -9,27 +9,10 @@ import { suite } from './suite';
 // eslint-disable-next-line @typescript-eslint/no-require-imports,@typescript-eslint/no-var-requires
 const tablemark = require('tablemark');
 
-const SUCCESS = '✅';
-const FAILURE = '❌';
-const NA = 'N/A';
-
-/**
- * Determine whether or not a test case should be excluded from a specific language.
- * An exclusion happens either when the entire language binding is excluded, or a specific test case.
- *
- * @param test the test.
- * @param suite the suite.
- * @param language the language.
- */
-function isExcluded(
-  testCase: schema.TestCase,
-  suite: schema.Suite,
-  language: string,
-) {
-  const testExcluded = !!testCase.exclusions?.[language];
-  const bindingExcluded = !!suite.exclusions?.[language];
-  return testExcluded || bindingExcluded;
-}
+const SUCCESS = '✅'; // test succeeded
+const FAILURE = '❌'; // test is failing
+const MISSING = ''; // test is not implemented yet
```

Comment:
Would add some text here, e.g `missing`, or a question mark emoji or something.

And should also be reflected at the bottom of https://github.com/aws/jsii/tree/main/tools/jsii-compliance which explains the report statuses.

Reply:
Changed to:

- 🟢 - Test passes for this language.
- ⚪ - Test is not applicable for this language.
- ⭕ - Test is not implemented (yet) for this language
- 🔴 - Test is failing for this language: missing feature or bug in language bindings.